### PR TITLE
[pt] Added rule ID:DAR_EXEMPLO_EXEMPLIFICAR

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya, @susanaboatto and @p-goulart 

Here is an academic/formal rule.

Follow the discussion here.
https://github.com/languagetool-org/languagetool/issues/10056

Thanks!

### CODE REVIEW

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml [hunk 1]
```diff
@@ -8479,6 +8479,42 @@ USA
...
+            <rule>
+                <pattern>
+                    <marker>
+                        <token inflected='yes'>dar</token>
+                        <token regexp='yes'>um|uns|algum|alguns</token>
+                        <token regexp='yes'>exemplos?</token>
+                    </marker>
+                    <token postag='_PUNCT|SPS00|CC|CS' postag_regexp='yes'/>
+                </pattern>
+                <message>Num contexto formal/académico, é preferível escrever &quot;exemplificar&quot;.</message>
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml:8494]: We need messages to be dialect-neutral. Can we remove `académico` from here, as it doesn't match the pt-BR spelling?

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml [hunk 2]
```diff
@@ -8479,6 +8479,42 @@ USA
...
+            <rule>
+                <pattern>
+                    <marker>
+                        <token inflected='yes'>dar</token>
+                        <token regexp='yes'>um|uns|algum|alguns</token>
+                        <token regexp='yes'>exemplos?</token>
+                        <token regexp='yes'>d[ao]s?</token>
+                    </marker>
+                </pattern>
+                <message>Num contexto formal/académico, é preferível escrever &quot;exemplificar&quot;.</message>
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml:8509]: Same here

**Comment by marcoagpinto** [comment]
@susanaboatto 

FIXED!

Sorry, I am sleeping.

I have fixed it.

Let me go to the other file.

**Comment by marcoagpinto** [comment]
@p-goulart 

It has conflicts, can you help?

Thanks!

**Comment by p-goulart** [comment]
@marcoagpinto, I can't resolve all your conflicts for you. Is [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line) any help?

**Comment by marcoagpinto** [comment]
@p-goulart 

I know you can't save me all the time.

I will create another pull request and press "merge" which is the easiest way of doing it (without stress)

**Pull request closed without merging**